src/Book.java

```diff
@@ -28,12 +28,12 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         Book book = (Book) o;
-        return age == book.age;
+        return age == book.age && Objects.equals(title, book.title);
```

Comment:
```suggestion
        return age == book.age && Objects.equals(title, book.title) && && Objects.equals(author, book.author);
```
автора тоже хорошо бы проверять